data_steward/tools/combine_ehr_rdr.py

```diff
@@ -46,10 +46,30 @@
 """
 import logging
 
+from jinja2 import Environment
+
+import app_identity
 import bq_utils
 import common
-from constants.tools import combine_ehr_rdr as combine_consts
 import resources
+from constants.tools import combine_ehr_rdr as combine_consts
+from utils import bq as bq
+
+jinja_env = Environment(
+    # help protect against cross-site scripting vulnerabilities
+    autoescape=True,
+    # block tags on their own lines
+    # will not cause extra white space
+    trim_blocks=True,
+    lstrip_blocks=True,
+    # syntax highlighting should be better
+    # with these comment delimiters
+    comment_start_string='--',
+    comment_end_string=' --')
+
+COPY_PID_RID_QUERY = jinja_env.from_string("""
+select * from `{{project}}.{{dataset}}.{{pid_rid_table}}`
```

Comment:
This query doesn't take into account the participant's age.  It copies the pid/rid table in its entirety.  Ideally, the copy will only copy data for those participants whose age is less than the maximum age.